mobile/modules/tasks/less/_task-details.less

```diff
@@ -0,0 +1,106 @@
+@import (once) "../../../less/_app.variables.less";
+
+.tasks-details {
+  &__progress-chart {
+    position: relative;
+    height: 12rem;
+    width: 12rem;
+    & path[stroke='#ffffff'] {
+      stroke: transparent;
+      stroke-width: 2;
+    }
+    & path[fill='none'] {
+      stroke: transparent;
+    }
+  }
+  &__sub-header {
+    height: 13.4rem;
+    background-color: @task-details-sub-header-color;
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    padding: 1.5rem 1.5rem 1.5rem 0;
+  }
+  &__formatted {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    left: 27%;
+    width: 50%;
+    font-size: 24px;
+    margin: 0 auto;
+    text-align: center;
+  }
+  &__text-wrapper {
+    display: flex;
+    padding-left: 1.8rem;
+    flex-flow: column nowrap;
+    justify-content: space-between;
+    height: 100%;
+  }
+  &__task-name {
+    font-size: 1.6rem;
+    letter-spacing: 0.2px;
+    line-height: 2.4rem;
+    padding-top: 1rem;
+  }
+  &__percentage {
+    font-size: 1.2rem;
+    color: @task-text-color;
+    font-family: @font-open-sans-semibold;
+  }
+  &__done {
+    padding-right: 1rem;
+  }
+  &__left {
+
+  }
+  &__list {
+    &-item {
+      position: relative;
+      display: flex;
+      flex-flow: row nowrap;
+      align-items: center;
+      padding: 0.7rem 1rem;
+      border-bottom: 1px solid @event-list-item-border-bottom;
+      background-color: @color-white;
+      font-family: @font-open-sans-semibold;
+      & .icon-1 { color: @priority-1-color; }
+      & .icon-2 { color: @priority-2-color; }
+      & .icon-3 { color: @priority-3-color; }
+      & .icon-4 { color: @priority-4-color; }
```

Comment:
no, we have classes not for color itself, but for some specific rule for task list